### PR TITLE
Docs: Fix rendering of Dimension and Color value viewers

### PR DESCRIPTION
**Pull request by reidbarber**
The interactive value viewers at https://react-spectrum.adobe.com/react-spectrum/styling.html#dimension-values and https://react-spectrum.adobe.com/react-spectrum/styling.html#color-values aren't rendering.

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

## 📝 Test Instructions:

Check https://reactspectrum.blob.core.windows.net/reactspectrum/2f616b52a6b9867d2992d630794e1aa45bf3805c/docs/react-spectrum/styling.html#dimension-values and https://reactspectrum.blob.core.windows.net/reactspectrum/2f616b52a6b9867d2992d630794e1aa45bf3805c/docs/react-spectrum/styling.html#color-values

### CODE REVIEW

File: packages/dev/docs/pages/react-spectrum/styling.mdx [hunk 1]
```diff
@@ -186,6 +187,8 @@ The list of Spectrum color values is visualized below. Use the picker to see how
 light and dark color schemes.
 
 ```tsx import
+import ReactDOM from 'react-dom';
```
Comment (devongovett) [inline packages/dev/docs/pages/react-spectrum/styling.mdx:190]: oh I think I changed this to import `react-dom/client`: https://github.com/adobe/react-spectrum/blob/9790448803156f5f1412819164b7750bcf6b48fe/packages/dev/parcel-transformer-mdx-docs/MDXTransformer.js#L474

So the later line needs to change to `ReactDOM.createRoot(document.getElementById('color-viewer')).render(<ColorViewer />)`

Comment (reidbarber) [reply packages/dev/docs/pages/react-spectrum/styling.mdx]: Both ways seemed to work, but I went ahead and updated it to this way.

Comment (snowystinger) [reply packages/dev/docs/pages/react-spectrum/styling.mdx]: did you push? i still see it importing from react-dom, not the client subpath

Comment (devongovett) [reply packages/dev/docs/pages/react-spectrum/styling.mdx]: ya, `createRoot` is the React 18 way. `ReactDOM.render` throws a warning in the console.

Comment (snowystinger) [reply packages/dev/docs/pages/react-spectrum/styling.mdx]: right, but i thought it was preferable to do
`import { createRoot } from "react-dom/client";`
?

wasn't aware that you could still get at it through just `react-dom`

Comment (reidbarber) [reply packages/dev/docs/pages/react-spectrum/styling.mdx]: I can't get the second one to render with `createRoot` for some reason. Looking into it.

Comment (reidbarber) [reply packages/dev/docs/pages/react-spectrum/styling.mdx]: Just updated, the culprit was a missing semicolon after the first render statement.

**Review by snowystinger — APPROVED** [review]

**Review by iamwillpowell — APPROVED** [review]

**Pull request merged**